Resources/public/templates/translateaction.hbt

```diff
@@ -26,9 +26,9 @@
                         <a class="ez-contenttranslation-edit-link" href="{{ path "editContent" id=../content.id languageCode=this }}">
                             {{translate 'translateaction.edit' 'bar'}}
                         </a>
-                        {{#if ../canDeleteTranslations}}
-                        <a class="ez-contenttranslation-delete-link ez-font-icon" data-translation="{{.}}" href="{{ path "editContent" id=../content.id languageCode=this }}"></a>
-                        {{/if}}
+                        {{#notEqual ../content.mainLanguageCode this }}
+                            <a class="ez-contenttranslation-delete-link ez-font-icon" data-translation="{{ this }}" href="{{ path "editContent" id=../content.id languageCode=this }}"></a>
```

Comment:
Is there any reason why you changed `{{ . }}` into `{{ this }}`?

Reply:
It's better to read, as `this` is used in every other place for this value. I can revert it.

Reply:
I was just curious. For me, it can stay.